filters/imagefilter.go

```diff
@@ -192,6 +194,10 @@ func transformImage(image *bimg.Image, opts *bimg.Options) ([]byte, error) {
 }
 
 func applyDefaults(o *bimg.Options) *bimg.Options {
+	val, exists := os.LookupEnv("STRIP_METADATA")
```

Comment:
Isn't os.LookupEnv() a syscall? If so, can we move this to some other place as here it runs for each request which might be too much?

Reply:
@rgritti maybe it could be passed as an option to the finalizeResponse filter constructor